api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -77,12 +80,14 @@
   private final WorkspaceService workspaceService;
 
   private static int NO_OF_PREVIEW_ROWS = 20;
-  private static int PREVIEW_RETRY_LIMIT = 2;
   // See https://cloud.google.com/appengine/articles/deadlineexceedederrors for details
   private static long APP_ENGINE_HARD_TIMEOUT_MSEC_MINUS_FIVE_SEC = 55000l;
   private static String CONCEPT_SET = "conceptSet";
   private static String COHORT = "cohort";
 
+  private static final String dateFormatString = "yyyy/MM/dd HH:mm:ss";
```

Comment:
Silly / newbie Java naming question: what's the difference between these constants and the constants above that makes them differently-cased?

Reply:
Renamed already :D